eve-api-data/src/main/java/com/eveonline/api/config/RepositoryConfiguration.java

```diff
@@ -1,15 +1,15 @@
 package com.eveonline.api.config;
 
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.orm.jpa.EntityScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
+import com.eveonline.api.repo.ApiDataRepositoryPackage;
+
 @Configuration
-@EnableAutoConfiguration
 @EntityScan(basePackages = {"com.eveonline.api"})
```

Comment:
I wanted to remove this but you actually showed me something here.  I'd never seen this annotation before.  However... with the way that repositories work with spring-data-jpa in that they're defined with their domain objects in the generics there is really no need for this annotation.  Since the repository relies on the domain object then spring-data-jpa add's it to the persistence context.